charts/testkube-operator/templates/executor.testkube.io_executors.yaml

```diff
@@ -4,6 +4,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.8.0
+  creationTimestamp: null
```

Comment:
i don't think we need it

Reply:
this was probably auto-generated by controller-gen